proto/blueprint.proto

```diff
@@ -214,8 +214,27 @@ message List {
   string ad_unit = 16;
 
   optional google.protobuf.Timestamp last_updated_date = 17;
+
+  optional Header header = 18;
 }
 
+
+message Header {
+  string header_text = 1;
+  optional MyGuardianFollow my_guardian_follow = 2;
+  optional string description = 3;
+  HeaderType alignment = 4;
+  optional Branding branding = 5;
```

Comment:
The message `List` has a property `branding` too.  Are there scenarios they would be different?  Can we use the `branding` there?

Reply:
Yeah, there is a bit of a duplication here. 

The idea behind it was to make the `Header` message atomic and with all the fields in order to be able to render it in different contexts.